2020-11-09T08:41:48.8336240+01:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-11-09T08:41:48.8431350+01:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2020-11-09T08:41:48.8436010+01:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-11-09T08:41:48.8436740+01:00  [INF] Hosting environment: "Development" (c3307c92)
2020-11-09T08:41:48.8437270+01:00  [INF] Content root path: "/Users/camillacarlsen/Documents/GitHub/KundeserviceModul/KundeserviceModul" (b5d60022)
2020-11-09T10:44:24.8994580+01:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-11-09T10:44:24.9144030+01:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2020-11-09T10:44:24.9148930+01:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-11-09T10:44:24.9149590+01:00  [INF] Hosting environment: "Development" (c3307c92)
2020-11-09T10:44:24.9150250+01:00  [INF] Content root path: "/Users/camillacarlsen/Documents/GitHub/KundeserviceModul/KundeserviceModul" (b5d60022)
2020-11-09T10:46:35.8804840+01:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-11-09T10:46:35.9293710+01:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-11-09T10:46:35.9678310+01:00  [INF] Hosting environment: "Development" (c3307c92)
2020-11-09T10:46:36.0214100+01:00  [INF] Content root path: "/Users/camillacarlsen/Documents/GitHub/KundeserviceModul/KundeserviceModul" (b5d60022)
2020-11-09T10:50:15.6688700+01:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-11-09T10:50:15.7207160+01:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-11-09T10:50:15.7315620+01:00  [INF] Hosting environment: "Development" (c3307c92)
2020-11-09T10:50:15.7370170+01:00  [INF] Content root path: "/Users/camillacarlsen/Documents/GitHub/KundeserviceModul/KundeserviceModul" (b5d60022)
2020-11-09T10:51:37.1709970+01:00 0HM44IJE7EAVH:00000001 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.AggregateException: One or more errors occurred. (The remote party closed the WebSocket connection without completing the close handshake.)
 ---> System.Net.WebSockets.WebSocketException (0x80004005): The remote party closed the WebSocket connection without completing the close handshake.
 ---> Microsoft.AspNetCore.Connections.ConnectionResetException: Connection reset by peer
 ---> System.Net.Sockets.SocketException (54): Connection reset by peer
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.Internal.SocketAwaitableEventArgs.<GetResult>g__ThrowSocketException|7_0(SocketError e)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.Internal.SocketAwaitableEventArgs.GetResult()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.Internal.SocketConnection.ProcessReceives()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.Internal.SocketConnection.DoReceive()
   --- End of inner exception stack trace ---
   at System.IO.Pipelines.PipeCompletion.ThrowLatchedException()
   at System.IO.Pipelines.Pipe.GetReadResult(ReadResult& result)
   at System.IO.Pipelines.Pipe.GetReadAsyncResult()
   at System.IO.Pipelines.Pipe.DefaultPipeReader.GetResult(Int16 token)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.DuplexPipeStream.ReadAsyncInternal(Memory`1 destination, CancellationToken cancellationToken)
   at System.Net.Security.SslStream.<FillBufferAsync>g__InternalFillBufferAsync|215_0[TReadAdapter](TReadAdapter adap, ValueTask`1 task, Int32 min, Int32 initial)
   at System.Net.Security.SslStream.ReadAsyncInternal[TReadAdapter](TReadAdapter adapter, Memory`1 buffer)
   at System.IO.Pipelines.StreamPipeReader.ReadAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpRequestStream.ReadAsyncInternal(Memory`1 buffer, CancellationToken cancellationToken)
   at System.Net.WebSockets.ManagedWebSocket.EnsureBufferContainsAsync(Int32 minimumRequiredBytes, CancellationToken cancellationToken, Boolean throwOnPrematureClosure)
   at System.Net.WebSockets.ManagedWebSocket.ReceiveAsyncPrivate[TWebSocketReceiveResultGetter,TWebSocketReceiveResult](Memory`1 payloadBuffer, CancellationToken cancellationToken, TWebSocketReceiveResultGetter resultGetter)
   at System.Net.WebSockets.ManagedWebSocket.ReceiveAsyncPrivate[TWebSocketReceiveResultGetter,TWebSocketReceiveResult](Memory`1 payloadBuffer, CancellationToken cancellationToken, TWebSocketReceiveResultGetter resultGetter)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PumpWebSocket(WebSocket source, WebSocket destination, Int32 bufferSize, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.AcceptProxyWebSocketRequest(HttpContext context, Uri destinationUri, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-09T10:51:37.2393620+01:00 0HM44IJE7EAVH:00000001 [WRN] The response has already started, the error page middleware will not be executed. (e1dd4c0d)
2020-11-09T10:51:37.4161200+01:00 0HM44IJE7EAVH:00000001 [ERR] Connection id ""0HM44IJE7EAVH"", Request id ""0HM44IJE7EAVH:00000001"": An unhandled exception was thrown by the application. (560e7d32)
System.AggregateException: One or more errors occurred. (The remote party closed the WebSocket connection without completing the close handshake.)
 ---> System.Net.WebSockets.WebSocketException (0x80004005): The remote party closed the WebSocket connection without completing the close handshake.
 ---> Microsoft.AspNetCore.Connections.ConnectionResetException: Connection reset by peer
 ---> System.Net.Sockets.SocketException (54): Connection reset by peer
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.Internal.SocketAwaitableEventArgs.<GetResult>g__ThrowSocketException|7_0(SocketError e)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.Internal.SocketAwaitableEventArgs.GetResult()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.Internal.SocketConnection.ProcessReceives()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.Internal.SocketConnection.DoReceive()
   --- End of inner exception stack trace ---
   at System.IO.Pipelines.PipeCompletion.ThrowLatchedException()
   at System.IO.Pipelines.Pipe.GetReadResult(ReadResult& result)
   at System.IO.Pipelines.Pipe.GetReadAsyncResult()
   at System.IO.Pipelines.Pipe.DefaultPipeReader.GetResult(Int16 token)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.DuplexPipeStream.ReadAsyncInternal(Memory`1 destination, CancellationToken cancellationToken)
   at System.Net.Security.SslStream.<FillBufferAsync>g__InternalFillBufferAsync|215_0[TReadAdapter](TReadAdapter adap, ValueTask`1 task, Int32 min, Int32 initial)
   at System.Net.Security.SslStream.ReadAsyncInternal[TReadAdapter](TReadAdapter adapter, Memory`1 buffer)
   at System.IO.Pipelines.StreamPipeReader.ReadAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpRequestStream.ReadAsyncInternal(Memory`1 buffer, CancellationToken cancellationToken)
   at System.Net.WebSockets.ManagedWebSocket.EnsureBufferContainsAsync(Int32 minimumRequiredBytes, CancellationToken cancellationToken, Boolean throwOnPrematureClosure)
   at System.Net.WebSockets.ManagedWebSocket.ReceiveAsyncPrivate[TWebSocketReceiveResultGetter,TWebSocketReceiveResult](Memory`1 payloadBuffer, CancellationToken cancellationToken, TWebSocketReceiveResultGetter resultGetter)
   at System.Net.WebSockets.ManagedWebSocket.ReceiveAsyncPrivate[TWebSocketReceiveResultGetter,TWebSocketReceiveResult](Memory`1 payloadBuffer, CancellationToken cancellationToken, TWebSocketReceiveResultGetter resultGetter)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PumpWebSocket(WebSocket source, WebSocket destination, Int32 bufferSize, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.AcceptProxyWebSocketRequest(HttpContext context, Uri destinationUri, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
2020-11-09T10:59:10.1854050+01:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-11-09T10:59:10.1983420+01:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2020-11-09T10:59:10.2011140+01:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-11-09T10:59:10.2012140+01:00  [INF] Hosting environment: "Development" (c3307c92)
2020-11-09T10:59:10.2012740+01:00  [INF] Content root path: "/Users/camillacarlsen/Documents/GitHub/KundeserviceModul/KundeserviceModul" (b5d60022)
2020-11-09T11:09:11.2658720+01:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-11-09T11:09:11.3006360+01:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2020-11-09T11:09:11.3016550+01:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-11-09T11:09:11.3017260+01:00  [INF] Hosting environment: "Development" (c3307c92)
2020-11-09T11:09:11.3017830+01:00  [INF] Content root path: "/Users/camillacarlsen/Documents/GitHub/KundeserviceModul/KundeserviceModul" (b5d60022)
2020-11-09T11:13:55.4186300+01:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-11-09T11:13:55.4892990+01:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2020-11-09T11:13:55.4896700+01:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-11-09T11:13:55.4897210+01:00  [INF] Hosting environment: "Development" (c3307c92)
2020-11-09T11:13:55.4897680+01:00  [INF] Content root path: "/Users/camillacarlsen/Documents/GitHub/KundeserviceModul/KundeserviceModul" (b5d60022)
2020-11-09T12:21:54.7549000+01:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-11-09T12:21:54.7633820+01:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2020-11-09T12:21:54.7642640+01:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-11-09T12:21:54.7643550+01:00  [INF] Hosting environment: "Development" (c3307c92)
2020-11-09T12:21:54.7644080+01:00  [INF] Content root path: "/Users/camillacarlsen/Documents/GitHub/KundeserviceModul/KundeserviceModul" (b5d60022)
2020-11-09T12:24:09.6738120+01:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-11-09T12:24:09.6935890+01:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2020-11-09T12:24:09.6971250+01:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-11-09T12:24:09.6972460+01:00  [INF] Hosting environment: "Development" (c3307c92)
2020-11-09T12:24:09.6972930+01:00  [INF] Content root path: "/Users/camillacarlsen/Documents/GitHub/KundeserviceModul/KundeserviceModul" (b5d60022)
